### DISCUSSION TITLE
Buggy indention in diffs with Firefox on Linux and Android

### DISCUSSION THREAD

**Discussion by iluminat23** [community/community#10032] [Projects and Issues]
The indention in diffs are wrong when using Firefox on Linux or Android.
Firefox Linux:
![gh indention issue](https://github.com/iluminat23/gh-indention-bug/raw/master/gh-indention-issue.png)

Chrome Linux:
![gh indention correct](https://raw.githubusercontent.com/iluminat23/gh-indention-bug/master/gh-indention-correct.png)

This is the test-repo I created for this:
https://github.com/iluminat23/gh-indention-bug/commit/4855c776a9a07ac55b0e79e6520567a2f40f7e60#diff-7842e7ce88f15c3ffd30c2437aced8764f9b62a009c8e15ffc3b045d0b834891

**Comment by nbuchwitz** [comment]
I have observed the same behaviour running Fedora 35 with Firefox 95.0.2. With Windows 10 and Firefox 95.0.2 everything looks as expected.
